src/bin/__test__/cli.test.ts

```diff
@@ -0,0 +1,37 @@
+import run from '../cli'
+
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
+test('cli prints help', () => {
+  const logSpy = jest.spyOn(global.console, 'log').mockImplementation(jest.fn())
+  let exitCode
+  //@ts-expect-error never type can't work here
+  const exitSpy = jest.spyOn(process, 'exit').mockImplementation((code?: number): never => {
+    exitCode = code
+  })
+  run(['/usr/bin/env/node', 'source-map-upload', '--help'])
+  expect(logSpy).toHaveBeenCalled()
+  expect(logSpy).toHaveBeenCalledWith(expect.stringContaining('bugsnag-source-maps <command>'))
+  expect(exitSpy).toHaveBeenCalled()
+  expect(exitCode).toBe(0)
```

Comment:
Instead of creating a closure on exitCode you could instead consider:

```
jest.spyOn(process, 'exit')
[...]
expect(exitSpy).toHaveBeenCalledWith(0)
```